test-cases/gutenberg/embed.md

```diff
@@ -0,0 +1,88 @@
+# Embed Block - Test Cases
+##### TC001
+
+**Inline previews are rendered correctly for the top 5 most-used providers**
+
+#### **Precondition**
+
+Perform these tests for each of the top 5 most-used providers below. 
+ 1. YouTube
+ 2. Twitter
+ 3. Instagram
```

Comment:
I'm wondering if we should warn that Instagram only works in WP.com sites or self-hosted sites connected to Jetpack, wdyt?

Reply:
Done 7b44afe